src/elisp/treemacs-impl.el

```diff
@@ -886,10 +885,13 @@ PROJECT Project Struct"
     (treemacs-goto-file-node path project))
    ((eq :custom (car path))
     (treemacs--goto-custom-top-level-node path))
-   (t
-    (treemacs--goto-custom-dir-node path))))
+   ((stringp (car path))
+    (treemacs--goto-custom-dir-node path))
+   (t 
+    (treemacs--goto-custom-project-node path))))
 
 (defun treemacs-find-file-node (path &optional project)
+  (declare (debug t))
```

Comment:
You've put that debug above the eldoc. Looks like I need to look at what `make lint` is doing, this is the kind of stuff that should be caught by travis.

Reply:
I wasn't supposed to commit the declaratio at all, removed now.